rust-ifcfg-devname.spec

```diff
@@ -72,6 +74,8 @@ use the "default" feature of the "%{crate}" crate.
 %install
 %cargo_install
 
+install -m 0755 -d $(DESTDIR)$(libdir)/udev
```

Comment:
It is not enough just to create the destination directory. There two problems, first, we also need to install actual `ifcfg-devname` after creating this directory. Second, `ifcfg-devname` is now installed into `/bin` which is default for binary crates. Probably the easiest solution will be adding `Makefile` that would have `install` target that installs the binary into correct location.